policies/secpolicy.html

```diff
@@ -12,7 +12,7 @@
 	  <header>
 	    <h2>Security Policy</h2>
 	    <h5>
-	      Last modified 16th May 2018
+	      Last modified 29th November 2018
```

Comment:
When performing the commit please update this to the closing date of the policy change vote.